deployment/terraform/destroy.go

```diff
@@ -20,6 +20,7 @@ func (t *Terraform) Destroy() error {
 		"-var", fmt.Sprintf("app_instance_type=%s", t.config.AppInstanceType),
 		"-var", fmt.Sprintf("agent_instance_count=%d", t.config.AgentInstanceCount),
 		"-var", fmt.Sprintf("agent_instance_type=%s", t.config.AgentInstanceType),
+		"-var", fmt.Sprintf("es_instance=%t", t.config.HasElasticSearch()),
```

Comment:
I think you also need to pass vpc id and everything else (after you remove the default values). This is what I got when I tried to destroy

```
020-06-05T00:04:01.948+0530	info	terraform/engine.go:81	
2020-06-05T00:04:01.986+0530	info	terraform/engine.go:81	var.vpc
2020-06-05T00:04:01.986+0530	info	terraform/engine.go:81	  Enter a value: 
```

While it did go ahead and destroyed everything, it would be good to avoid this.

Reply:
Ah wait, will it destroy the entire VPC if I pass the vpc id ? Hmm .. we might need to create a VPC from scratch then. Something for later I guess.

Reply:
Actually, it shouldn't. Because we aren't creating the VPC. Just creating stuff inside it. So should be fine.

Reply:
AFAIK the destroy command only destroys the infrastructure created by terraform so passing the id should be fine and it shouldn't destroy the VPC